src/environment.cpp

```diff
@@ -397,8 +397,11 @@ void fillRadiusBlock(v3s16 p0, s16 r, std::set<v3s16> &list)
 	for(p.Y=p0.Y-r; p.Y<=p0.Y+r; p.Y++)
 	for(p.Z=p0.Z-r; p.Z<=p0.Z+r; p.Z++)
 	{
-		// Set in list
-		list.insert(p);
+		// limit to a sphere
+		if (p.getDistanceFrom(p0) <= r) {
```

Comment:
There is no problem at all in adding braces (i.e. making a multiline statement) where they are not needed as long as the surrounding code is consistent. Many style guides actually stipulate that braces should always be used (I don't necessarily agree this is necessary but I don't disagree either). For readability then I prefer braces as in the first example if there is a comment involved. Also it makes things obvious (e.g. @paramat it's not counted as a line because a: )the pre-processor strips comments before actual compilation; and b) it's not the number of lines that are important, it's the number of *statements*.